docker-plugin/src/main/java/com/nirima/jenkins/plugins/docker/DockerTemplateBase.java

```diff
@@ -110,16 +111,32 @@ public DockerTemplateBase(String image,
 
     protected Object readResolve() {
         if (volumesFrom != null) {
```

Comment:
Are you kidding me?

What for you need two ifs if second if also checks first?
And why you asking me ok, or not, if you don't listen me?